business_register/converter/fop.py

```diff
@@ -49,83 +54,158 @@ def create_new_fop(self, hash_code, fullname, address, status, registration_date
             contact_info=contact_info,
             vp_dates=vp_dates,
             authority=authority
-            )
+        )
         return fop
-    
-    #putting all kveds into a list
+
+    # putting all kveds into a list
     def add_fop_kveds_to_list(self, fop_kveds, hash_code):
         for activity in fop_kveds:
-            info = activity.xpath('ACTIVITY_KIND/CODE')
+            info = activity.xpath('CODE')
             if not info:
-                return
+                continue
             kved_code = info[0].text
             kved = self.get_kved_from_DB(kved_code)
-            primary = True if activity.xpath('ACTIVITY_KIND/PRIMARY')[0].text == "так" else False
+            primary = True if activity.xpath('PRIMARY')[0].text == "так" else False
```

Comment:
...??? redundant code
```suggestion
            primary = activity.xpath('PRIMARY')[0].text == "так"
```

Reply:
)) done